packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -77,6 +83,19 @@ export const CloudPulseResourcesSelect = React.memo(
       return resources && resources.length > 0 ? resources : [];
     }, [resources]);
 
+    // Maximum resource selection limit is fetched from launchdarkly
+    const maxResourceSelectionLimit = React.useMemo(() => {
+      const obj = flags.aclpResourceTypeMap?.find(
+        (item: CloudPulseResourceTypeMapFlag) =>
+          item.serviceType === resourceType
```

Comment:
```suggestion
        (item) => item.serviceType === resourceType
```

no need of type check, typescript will take care it